holoviews/plotting/bokeh/plot.py

```diff
@@ -303,7 +310,7 @@ def sync_sources(self):
                         renderer.update(data_source=new_source)
                     else:
                         renderer.update(source=new_source)
-                    if hasattr(renderer, 'view'):
+                    if not bokeh3 and hasattr(renderer, 'view'):
```

Comment:
Not needed or has `Renderer.view` disappeared?

Reply:
CDSView does not contain source anymore. From the migration guide:

![image](https://user-images.githubusercontent.com/19758978/214241821-9d938a8c-9020-478b-b208-1083a94f01cf.png)